src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1108,6 +1115,7 @@ int sockinfo_udp::setsockopt(int __level, int __optname, __const void *__optval,
 				si_udp_logdbg("found UDP_MAP_ADD socket fd for port %d. fd is %d", ntohs(port_socket.port), port_socket.fd);			
 				m_port_map.push_back(port_socket);
 			}
+			m_mapped = true;
```

Comment:
The setting/resetting of `m_mapped` affects the value of  `m_rx_packet_processor` when it's set in `attach_receiver()` (before starting to receive data).
Setting `m_mapped` in `sockinfo_udp::setsockopt()` may occur any time - also during data flow. If this is called any time after `attach_receiver()` it will have no effect on `m_rx_packet_processor`.
Please verify if this is OK.

BTW, `m_mapped` is set during the handling of `UDP_MAP_ADD` / `UDP_MAP_REMOVE` `setsockopt` options which exist in the code since it's 1st git commit. By their definition there's a comment: `/* For MCD */`.
Any idea what this is about?

@rosenbaumalex, @alexvm - any take on this?